internal/project/common/image.go

```diff
@@ -21,10 +23,10 @@ type Image struct {
 
 	meta *meta.Options
 
-	DroneExtraEnvironment map[string]string `yaml:"droneExtraEnvironment"`
```

Comment:
this is an incompatible change... I know it was named poor way, but should we provide backwards compatibility here instead of renaming?

Reply:
there's only some projects using it, I was thinking of updating those, makes easier, since we only use this internally